themes/crisp/translations/pt_BR.json

```
@@ -88,18 +88,18 @@
     "views.classification.class.c": "Nota C",
     "views.classification.class.c.desc": "Os termos de serviço são ok mas algumas questões merecem a sua consideração.",
     "views.classification.class.d": "Nota D",
-    "views.classification.class.d.desc": "Os termos de serviço são muito injustos ou há algumas questões importantes que precisam da sua atenção.",
+    "views.classification.class.d.desc": "Os termos de serviço são muito injustos ou existem algumas questões importantes que precisam da sua atenção.",
     "views.classification.class.e": "Nota E",
     "views.classification.class.e.desc": "Os termos de serviço levantam preocupações muito sérias.",
-    "views.classification.class.none": "Ainda sem nota",
+    "views.classification.class.none": "Ainda não avaliado",
     "views.classification.class_none_desc": "Nós ainda não revisamos os termos.",
     "views.classification.container.text.1": "Os termos de serviço são revisados por contribuidores e divididos em pequenos pontos que podemos discutir, comparar e, enfim, atribuir uma nota com uma insígnia:",
-    "views.classification.container.text.2": "Uma vez que o serviço tem insígnias suficientes para avaliar justeza de seus termos de serviço para os usuários, uma classe é atribuída automaticamente, ponderando as pontuações médias.",
-    "views.classification.container.text.3": "Neste momento, você irá perceber que não há muitos serviços com uma classe atribuída. Isso é porque nós precisamos de mais dados e mais revisões antes que possamos começar a atribui-las. Além disso, ainda estamos experimentado como aplicar as classes.",
+    "views.classification.container.text.2": "Uma vez que o serviço tem insígnias suficientes para avaliar a justeza de seus termos de serviço para os usuários, uma classe é atribuída automaticamente, ponderando as pontuações médias.",
+    "views.classification.container.text.3": "Neste momento, você irá perceber que não existem muitos serviços com uma classe atribuída. Isso é porque nós precisamos de mais dados e mais revisões antes que possamos começar a atribui-las. Além disso, ainda estamos experimentado como aplicar as classes.",
     "views.classification.jumbotron.header": "Classificação",
-    "views.classification.jumbotron.text": "O ToS;DR visa criar um processo transparente e revisado por pares e analisar Termos de Serviço e Políticas de Privacidade para criar uma avaliação da Nota A à Nota E. ",
+    "views.classification.jumbotron.text": "O ToS;DR visa criar um processo transparente e fortemente revisado para avaliar e analisar Termos de Serviço e Políticas de Privacidade a fim de criar uma avaliação que parte da Nota A até a Nota E. ",
```

```suggestion
    "views.classification.jumbotron.text": "O ToS;DR visa criar um processo transparente e revisado por pares para avaliar e analisar Termos de Serviço e Políticas de Privacidade a fim de criar uma avaliação que parte da Nota A até a Nota E. ",
```

O *peer-review* é um processo usado no meio acadêmico e em publicações científicas visando uma maior confiabilidade. Ele é frequentemente traduzido para português como [revisão por pares](https://pt.wikipedia.org/wiki/Revis%C3%A3o_por_pares).

---

Sim, estou ciente, é que pensei em trazer de maneira mais popular, _peer-review_ no Brasil é um conceito muito específico da nossa área, mas acho que aqui a ideia é falar de maneira mais técnica mesmo né?

---

Acho que é assim em qualquer país, não só no Brasil. Então optei por manter a mesma linguagem.

---

Então show, perfeito